### PR TITLE
Add: `DevOps` topic

### PR DESCRIPTION
**Pull request by snxx-lppxx**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

Added a topic from TODO List.

### CODE REVIEW

**Comment by Rob122783** [comment]
Liking it

**Review by Rob122783 — APPROVED** [review]

**Comment by Rob122783** [comment]
> ### Thank you for contributing! Please confirm this pull request meets the following requirements:
> 
> - [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
> - [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).
> 
> ### Which change are you proposing?
> 
>   - [x] Curating a new topic or collection
> 
> ---------------------------------------------------------------------
> 
> 
> ### Editing an existing topic or collection
> 
> I'm suggesting these edits to an existing topic or collection:
> - [x] Image (and my file is `*.png`, square, dimensions 288x288)
> - [x] Content (and my changes are in `index.md`)
> 
> Added a topic from TODO List.
>

**Review by Rob122783 — APPROVED** [review]

**Comment by sina6565** [comment]
@sina6565

**Review by Beheratapas — APPROVED** [review]

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

File: topics/devops/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+aliases: dev-ops
+display_name: DevOps
+logo: devops.png
+related: ci, cd
+short_description: DevOps is a set of practices that combines software development (Dev) and IT operations (Ops).
+topic: devops
+wikipedia_url: https://en.wikipedia.org/wiki/DevOps
+---
+DevOps is a set of practices that combines software development (Dev) and IT operations (Ops). It aims to shorten the systems development life cycle and provide continuous delivery with high software quality.
```
Comment (kenyonj) [inline topics/devops/index.md:10]:
```suggestion
short_description: DevOps is a set of practices that combine software development (Dev) and IT operations (Ops).
topic: devops
wikipedia_url: https://en.wikipedia.org/wiki/DevOps
---
DevOps is a set of practices that combine software development (Dev) and IT operations (Ops). It aims to shorten the systems development life cycle and provide continuous delivery with high software quality.
```

**Pull request merged**